sonic-xcvrd/xcvrd/dom/utilities/dom_sensor/db_utils.py

```diff
@@ -0,0 +1,140 @@
+import re
+from xcvrd.dom.utilities.db.utils import DBUtils
+from xcvrd.dom.utilities.dom_sensor.utils import DOMUtils
+from swsscommon import swsscommon
+
+
+class DOMDBUtils(DBUtils):
+    """
```

Comment:
@mihirpat1 name of the STATE_DB table in code comment? Can you do this for other db utils/tables?